wordpress-mobile/WordPress-iOS
# Likes: use WPKit endpoint changes

Ref:  #15662
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/389

To test:
As the new data from WPKit is not yet used in WPiOS, just be sure the likes list still appears.
- Go to Notifications > Likes.
- Select various notifications.
- Verify the likes list is displayed.

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete.

3. What automated tests I added (or what prevented me from doing so)
Updated relevant Likes tests and WPKit Likes tests.

PR submission checklist:
